src/ray/common/id.h

```diff
@@ -27,6 +27,86 @@ class JobID;
 /// A helper function that get the `DriverID` of the given job.
 WorkerID ComputeDriverIdFromJob(const JobID &job_id);
 
+enum class ObjectType : uint8_t {
+  PUT_OBJECT,
+  RETURN_OBJECT,
+};
+
+enum class TransportType : uint8_t {
+  STANDARD,
+  DIRECT_ACTOR_CALL,
+};
```

Comment:
can we use the transport definitions in core_worker to prevent further inconsistencies?

Reply:
I don't think `id` module should depends on the `core_worker` module. How about define them in a parent module like `common` ?

Reply:
That makes sense. On the other side, I would prefer to keep the exact definitions of transport type inside the `core_worker` module, if this information is only needed by `core_worker` itself.  The current code can remain as is, and this could be addressed later I think.

Reply:
Fine. I left a TODO comment on this.

Reply:
Rather than having 2 separate definitions, I prefer to just using `int` for the transport type here.
Because the id module doesn't care about the transport type, it only needs to store type. Core worker can handle the conversion by itself.

Reply:
> Rather than having 2 separate definitions, I prefer to just using `int` for the transport type here.
> Because the id module doesn't care about the transport type, it only needs to store type. Core worker can handle the conversion by itself.


A nice tradeoff. Done!